# HG changeset patch
# User Rik <rik@octave.org>
# Date 1377820117 25200
#      Thu Aug 29 16:48:37 2013 -0700
# Node ID 1391dc953ae2a0b745ed5375153af306d2d077f9
# Parent  68ff3d60c48af997499d75492e1bc3f4a1a066cd
__patch__.m: Fix typo and close bug #34417.

* scripts/plot/private/__patch__.m: Change 'agrs' to 'args'.

diff --git a/scripts/plot/private/__patch__.m b/scripts/plot/private/__patch__.m
--- a/scripts/plot/private/__patch__.m
+++ b/scripts/plot/private/__patch__.m
@@ -24,16 +24,17 @@
 ## __patch__ (p, x, y, c)
 ## Create patch object from x and y with color c and parent p.
 ## Return handle to patch object.
 
 ## Author: Kai Habel
 
 function [h, failed] = __patch__ (p, varargin)
 
+  keyboard
   h = NaN;
   failed = false;
 
   is_numeric_arg = cellfun (@isnumeric, varargin);
 
   if (isempty (varargin))
     args = {"xdata", [0; 1; 0], "ydata", [1; 1; 0], "facecolor", [0, 0, 0]};
     args = setvertexdata (args);
@@ -143,17 +144,17 @@ function [h, failed] = __patch__ (p, var
             args{8} = "flat";
             args{9} = "cdata";
             args{10} = c;
           elseif (rows (c) == rows (x) && columns (c) == columns (x))
             ## Per-vertex color
             args{7} = "facecolor";
             args{8} = "interp";
             args{9} = "cdata";
-            agrs{10} = c;
+            args{10} = c;
           else
             error ("patch: color value not valid");
           endif
         else
           ## Color Vectors
           if (isempty (c))
             args{7} = "facecolor";
             args{8} = "interp";
